akka-http2-support/src/main/scala/akka/http/impl/engine/http2/TlsActorCopy.scala

```diff
@@ -0,0 +1,485 @@
+/**
+ * Copyright (C) 2015-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.impl.io
+
+import java.nio.ByteBuffer
+import javax.net.ssl.SSLEngineResult.HandshakeStatus
+import javax.net.ssl.SSLEngineResult.HandshakeStatus._
+import javax.net.ssl.SSLEngineResult.Status._
+import javax.net.ssl._
+
+import akka.actor._
+import akka.stream._
+import akka.stream.impl.FanIn.InputBunch
+import akka.stream.impl.FanOut.OutputBunch
+import akka.stream.impl._
+import akka.util.ByteString
+
+import scala.annotation.tailrec
+import akka.stream.TLSProtocol._
+
+import scala.util.control.NonFatal
+import scala.util.{ Failure, Success, Try }
+
+/**
+ * INTERNAL API.
+ */
+private[stream] object TLSActor {
```

Comment:
what has to change in this guy to not have to copy it?
Or is it simply the current not released Akka?